viam-cartographer_internal_test.go

```diff
@@ -428,6 +428,81 @@ func TestGetPointCloudMapEndpoint(t *testing.T) {
 	})
 }
 
+func setMockGetPointCloudFunc(
+	mock *cartofacade.Mock,
+	pc []byte,
+) {
+	mock.GetPointCloudMapFunc = func(
+		ctx context.Context,
+		timeout time.Duration,
+	) ([]byte, error) {
+		return pc, nil
+	}
+}
+
+func TestGetPointCloudMapEndpointModularizationV2Endpoint(t *testing.T) {
+	svc := &cartographerService{Named: resource.NewName(slam.API, "test").AsNamed()}
+	mockCartoFacade := &cartofacade.Mock{}
+
+	svc.cartofacade = mockCartoFacade
+	svc.modularizationV2Enabled = true
+
+	inputPointCloudMapBytes := []byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}
+	setMockGetPointCloudFunc(mockCartoFacade, inputPointCloudMapBytes)
+
+	t.Run("small chunk size success", func(t *testing.T) {
```

Comment:
Can we test the behavior when the response size is above & below the chunk limit with real-world files, i.e. without needing to modify the chunk size?  

Something like:
 `.artifact/data/viam-cartographer/outputs/viam-office-02-22-3/pointcloud/pointcloud_0.pcd` for a file below the 1 MB limit & 
`.artifact/data/viam-cartographer/outputs/viam-office-02-22-3/pointcloud/pointcloud_1.pcd` for a file over the limit.

Reply:
also, please have the test confirm that those files are indeed below & above the chunk size limit, respectively.